tools/apiview/emitters/typespec-apiview/package.json

```diff
@@ -53,19 +53,18 @@
     "dist/**",
     "!dist/test/**"
   ],
-  "dependencies": {
+  "peerDependencies": {
+    "@typespec/compiler": ">=0.40 <1.0",
+    "@typespec/versioning": ">=0.40 <1.0"
+  },
+  "devDependencies": {
     "@azure-tools/typespec-azure-core": "0.29.0",
```

Comment:
@tjprescott is devDependency correct for these? You no longer have any runtime dependencies.

Reply:
Based on feedback from @mikeharder he suggested making this dev dependencies since these packages are only used in the APIView tests. The hard dependencies are really just the ones listed as peer dependencies: compiler and verisoning.